projects/packages/my-jetpack/_inc/state/resolvers.js

```diff
@@ -64,6 +65,21 @@ const myJetpackResolvers = {
 			}
 		},
 
+	getChatAvailability:
+		() =>
+		async ( { dispatch } ) => {
+			dispatch.setChatAvailabilityIsFetching( true );
+
+			try {
+				dispatch.setChatAvailability(
+					await apiFetch( { path: REST_API_CHAT_AVAILABILITY_ENDPOINT } )
+				);
+				dispatch.setChatAvailabilityIsFetching( false );
+			} catch ( error ) {
+				dispatch.setChatAvailabilityIsFetching( false );
```

Comment:
Note: The other API failures in this package display a message on the `my-jetpack` dashboard indicating there was some sort of issue. For this one, I did not do that because failing quietly on a chat widget seemed better in this case